components/ServiceForm.vue

```diff
@@ -13,7 +13,7 @@
                   name="Name"
                   :rules="'required'"
                 >
-                  <v-text-field v-model="form.name" :error-messages="errors">
+                  <v-text-field v-model="form.name" :error-messages="errors"  v-on:keyup="slugifyName" v-on:change="slugifyName">
```

Comment:
You can use @keyup and @change here.. shorthand @RomuloLlabresJr

Reply:
@ismaeljhon i checked update code to use shorthand on keyup